content/posts/jellyfin-10-5-0.md

```diff
@@ -0,0 +1,170 @@
+
+Finally, several performance and responsiveness issues have been addressed. A rogue gamepad input loop is now properly handled, which prevents unnecessary repaintings by the browser and should improve performance. A previously-disabled fade-in effect for images has also been enabled, making loading into a new page look smoother. It is now also possible to upload artwork in WEBP format.
+
+## Server
+
+This cycle, our server team ported the code over to .NET Core 3.1. This move makes new features available to our developers, including support for ARM64 for Linux, compatibility with TLS v1.3 and better garbage collection on Docker, allowing for better memory usage when running in a container.
+
+A lot of code cleanup and refactoring was done, fixing a number issues and bringing some speed improvements. Among these, the following table, while a little technical, shows the impressive speed improvement between the previous implementation of our hex decoder and the one shipped in Jellyfin 10.5.0.
+
+<table style="width:100%">
+<thead>
+    <th>Method</th>
+    <th>Mean</th>
+    <th>Error</th>
+    <th>StdDev</th>
+    <th>Gen 0</th>
+    <th>Allocated</th>
+</thead>
+<tbody>
+<tr>
+    <td><b>Jellyfin 10.5.0</b></td>
+    <td style="text-align:right">10.81 μs</td>
+    <td style="text-align:right">0.079 μs</td>
+    <td style="text-align:right">0.074 μs</td>
+    <td style="text-align:right">0.1068</td>
+    <td style="text-align:right">336 B</td>
+</tr>
+<tr>
+    <td><b>Jellyfin 10.4.3</b></td>
+    <td style="text-align:right">23.57 μs</td>
+    <td style="text-align:right">0.055 μs</td>
+    <td style="text-align:right">0.051 μs</td>
+    <td style="text-align:right">11.2305</td>
+    <td style="text-align:right">35272 B</td>
+</tr>
+</tbody>
+</table>
+
+Another example of the speed improvements to the server this cycle is the following comparison between our previous alphanumeric comparator and the one shipped in Jellyfin 10.5.0, showing a reduction of the execution time of more than 50%!
+
+<table style="width:100%">
+<thead>
+    <th>Method</th>
+    <th>Mean</th>
+    <th>Error</th>
+    <th>StdDev</th>
+    <th>Gen 0</th>
+    <th>Allocated</th>
+</thead>
+<tbody>
+<tr>
+    <td><b>Jellyfin 10.5.0</b></td>
+    <td style="text-align:right">10.81 μs</td>
+    <td style="text-align:right">0.079 μs</td>
+    <td style="text-align:right">0.074 μs</td>
+    <td style="text-align:right">0.1068</td>
+    <td style="text-align:right">336 B</td>
+</tr>
+<tr>
+    <td><b>Jellyfin 10.4.3</b></td>
+    <td style="text-align:right">23.57 μs</td>
+    <td style="text-align:right">0.055 μs</td>
+    <td style="text-align:right">0.051 μs</td>
+    <td style="text-align:right">11.2305</td>
+    <td style="text-align:right">35272 B</td>
+</tr>
+</tbody>
+</table>
```

Comment:
This section is plain wrong. You're repeating a single benchmark and saying that it's two different things, as I commented you on Matrix. However, we still have no confirmation from @jellyfin/backend about what's beign benchmarked here.

Notice that two benchmarks give exact same results

Reply:
Indeed, I removed that part since we're not sure.